tests/Models/OrderStatusV2.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Contains the OrderStatus class.
+ *
+ * @copyright   Copyright (c) 2017 Attila Fulop
+ * @author      Attila Fulop
+ * @license     MIT
+ * @since       2017-10-05
```

Comment:
Please update the docblock meta tags when creating a new file so that it contains the author's name and the date of creation (`@since`)

Reply:
fixed it, I was doubting with the copyright but left it as it is